network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -114,17 +103,33 @@ def filter_entries_for_category(self, entries, context):
         # store the base category name
         context['terms'] = [category.name, ]
 
-        # then explicitly set the index page title and intro, making
+        # then explicitly set all the metadata that can be localized, making
         # sure to use the localized category for those fields:
         locale = get_locale_from_request(context['request'])
         try:
             localized_category = category.get_translation(locale)
         except ObjectDoesNotExist:
             localized_category = category
 
-        context['index_title'] = titlecase(f'{localized_category.name} {self.title}')
+        context['index_title'] = localized_category.title
```

Comment:
Let's make sure to fall back to `titlecase(f'{localized_category.name} {self.title}')` if `localized_category.title` is empty